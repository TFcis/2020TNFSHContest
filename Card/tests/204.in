10
2 3 6 7 1 4 9 5 10 8
5 4 9 3 7 2 1 10 6 8
93
