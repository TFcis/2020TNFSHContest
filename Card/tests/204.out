2 6 4 10 1 7 9 8 3 5
